### PR TITLE
Add test support

### PR DESCRIPTION
**Pull request by adiabramovitch**

### CODE REVIEW

**Comment by adiabramovitch** [comment]
@ofedoren Did I miss something? 
when I tried to run the tests I failed but in the same way, the hammer-CLI-foreman-host-reports failed to run as well. 
image

File: test/test_helper.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+ENV['TEST_API_VERSION'] = ENV['TEST_API_VERSION'] || '3.2'
```
Comment (ofedoren) [inline test/test_helper.rb]: Well, it's more like 3.3 now :)

File: test/data/3.2/foreman_api.json [hunk 2]
```diff
@@ -0,0 +1 @@
...":[]},{"name":"host_id","full_name":"host_id","description":"\n\u003cp\u003eID or name of host\u003c/p\u003e\n","required":true,"allow_nil":false,"allow_blank":false,"validator":"Must be a String","expected_type":"string","metadata":null,"show":true,"validations":[]},{"name":"id","full_name":"id","description":"\n\u003cp\u003eID of interface\u003c/p\u003e\n","required":true,"allow_nil":false,"allow_blank":false,"validator":"Must be a String","expected_type":"string","metadata":null,"show":true,"validations":[]}],"returns":[],"examples":["DELETE /api/hosts/host66/interfaces/2\n{\n  \"interface\": {}\n}\n200\n{\n  \"id\": 2,\n  \"mac\": \"00:33:45:ab:00:10\",\n  \"ip\": \"0.0.0.11\",\n  \"name\": null,\n  \"host_id\": 1,\n  \"subnet_id\": null,\n  \"domain_id\": null,\n  \"attrs\": {},\n  \"created_at\": \"2021-05-18T11:45:35.580Z\",\n  \"updated_at\": \"2021-05-18T11:45:35.580Z\",\n  \"provider\": null,\n  \"username\": null,\n  \"password\": null,\n  \"virtual\": false,\n  \"link\": true,\n  \"identifier\": \"eth28\",\n  \"tag\": \"\",\n  \"attached_to\": \"\",\n  \"managed\": true,\n  \"mode\": \"balance-rr\",\n  \"attached_devices\": \"\",\n  \"bond_options\": \"\",\n  \"primary\
```
Comment (ofedoren) [inline test/data/3.2/foreman_api.json]: Re-generate please, otherwise tests will fail.

**Review by ofedoren — CHANGES_REQUESTED** [review]
@adiabramovitch, you need to have locally `Gemfile.local` in you `hammer-cli-foreman-webhooks` checkout with something like:
```ruby
path '../' do
  gem 'hammer_cli'
  gem 'hammer_cli_foreman' 
end
```
And then if you run `bundle exec rake test` after `bundle install` or `bundle update`, it should work.

I've tested this PR with https://github.com/theforeman/hammer-cli-foreman-webhooks/pull/3 and I've noticed that all the tests fail. I guess you have corrupted `foreman_api.json` here. When I re-cached and put my JSON here it worked. Although 1 test still fails :/

File: test/data/3.2/foreman_api.json (hunk 2 above)
Comment (adiabramovitch) [reply test/data/3.2/foreman_api.json]: I regenerate but I don't think it works...

**Review by ofedoren — APPROVED** [review]
Thanks, @adiabramovitch! I've tested with https://github.com/theforeman/hammer-cli-foreman-webhooks/pull/3 and I have no other objections so far, so let's merge and see what happens :D

**Pull request merged**